tech/languages/julia/julia-installation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Julia
+subsection: julia
+section: tech-languages
+version: 1.7.3
+order: 1
+description: High-level, high-performance dynamic language for technical computing.
+---
+
+# Julia installation and usage
+
+## Julia installation
```

Comment:
I think this is duplicating the content here. As there is no complicated usage I'd simply do a single h1 header

```suggestion
# Julia installation
```

Alternatively, do another h2 header after the installation steps.